userapi/consumers/roomserver.go

```diff
@@ -408,6 +408,7 @@ func (s *OutputRoomEventConsumer) localRoomMembers(ctx context.Context, roomID s
 	req := &rsapi.QueryMembershipsForRoomRequest{
 		RoomID:     roomID,
 		JoinedOnly: true,
+		LocalOnly:  true,
```

Comment:
This was just a bug that we omitted this?

Reply:
`LocalOnly` was only added after we got push rules, so this may have been an oversight.